docs/charts/43/supplemental/language_groups.html

```diff
@@ -64,63 +64,63 @@
 <p><b>Caveats:</b> Only the wikidata containment for <a href='http://unicode.org/reports/tr35/#unicode_language_subtag'>valid language codes</a> is used.The containment data is not complete: if a language doesn't appear in the chart it could be an isolate, or just be missing data.The data doesn't completely match wikipedia’s; there are some patches for CLDR languages.</p>
 <table>
 	<tr><th class='source'>Language Group</th><th class='source'>Name</th><th class='source'>St.</th><th class='source'>Contained</th></tr>
-	<tr><td class='source'><a name='mul' href='#mul'>mul</a></td><td class='source'>All</td><td class='source'>🌲</td><td class='target'>afa “Afro-Asiatic”; aqa “Alacalufan”; aql “Algic”; tut “Altaic”; auf “Arauan”; awd “Arawakan”; art “Artificial”; aus “Australian”; aav “Austro-Asiatic”; map “Austronesian”; euq “Basque (family)”; cdd “Caddoan”; cau “Caucasian”; cba “Chibchan”; cpe “Creoles and pidgins, English based”; dra “Dravidian”; esx “Eskimo-Aleut”; hmx “Hmong-Mien”; hok “Hokan”; ine “Indo-European”; iro “Iroquoian”; jpx “Japanese (family)”; khi “Khoisan”; myn “Mayan”; xnd “Na-Dene”; nic “Niger-Kordofanian”; ssa “Nilo-Saharan”; omq “Oto-Manguean”; paa “Papuan”; sal “Salishan”; sgn “Sign Languages”; sit “Sino-Tibetan”; sio “Siouan”; tai “Tai”; tuw “Tungus”; tup “Tupi”; urj “Uralic”; azc “Uto-Aztecan”; wak “Wakashan”</td></tr>
+	<tr><td class='source'><a name='mul' href='#mul'>mul</a></td><td class='source'>All</td><td class='source'>🌲</td><td class='target'>afa “Afro-Asiatic”; aqa “Alacalufan”; aql “Algic”; tut “Altaic”; auf “Arauan”; awd “Arawakan”; art “Artificial”; aus “Australian”; aav “Austro-Asiatic”; map “Austronesian”; euq “Basque (family)”; btk “Batak (Indonesia)”; cdd “Caddoan”; cau “Caucasian”; cba “Chibchan”; cpe “Creoles and pidgins, English based”; dra “Dravidian”; esx “Eskimo-Aleut”; hmx “Hmong-Mien”; hok “Hokan”; ine “Indo-European”; iro “Iroquoian”; jpx “Japanese (family)”; khi “Khoisan”; myn “Mayan”; xnd “Na-Dene”; nic “Niger-Kordofanian”; ssa “Nilo-Saharan”; omq “Oto-Manguean”; paa “Papuan”; sal “Salishan”; sgn “Sign Languages”; sit “Sino-Tibetan”; sio “Siouan”; tai “Tai”; tuw “Tungus”; tup “Tupi”; urj “Uralic”; azc “Uto-Aztecan”; wak “Wakashan”</td></tr>
```

Comment:
it might be worth considering newlines instead of spaces following the semi-colons in this file, since html renders them the same and the diffs would be a lot easier to parse visually

Reply:
Excellent idea!